include/opw_kinematics/opw_kinematics.h

```diff
@@ -1,12 +1,11 @@
 #ifndef OPW_KINEMATICS_H
 #define OPW_KINEMATICS_H
 
-#include <Eigen/Dense>
-#include "opw_kinematics/opw_parameters.h"
+#include <Eigen/Dense>                      // IWYU pragma: export
```

Comment:
Eigen seems like a type you wouldn't want to export. If someone is using Eigen in their file, they should include it themselves and not rely on this header.

Reply:
fixed.